src/commands/app/undeploy.js

```diff
@@ -59,7 +60,11 @@ class Undeploy extends BaseCommand {
 
       // undeploy
       if (!flags.static) {
-        await scripts.undeployActions()
+        if (fs.existsSync('actions/')) {
```

Comment:
@sandeep-paliwal can we change this to fs.existsSync('manifest.yml') so we are aligned with app-scripts and https://github.com/adobe/aio-cli-plugin-app/pull/171